### PR TITLE
feat(sdk/python): add the required files for packing package

### PR DESCRIPTION
**Pull request by lianxmfor**
this PR is for the packing packages.  follow document: https://packaging.python.org/tutorials/packaging-projects/

### CODE REVIEW

File: sdk/python/setup.cfg [hunk 1]
```diff
@@ -0,0 +1,24 @@
+[metadata]
+name = oomstore
+version = 0.0.1
+author = oom
+author_email = author@example.com
```
Comment (lianxmfor) [inline sdk/python/setup.cfg:5]: I'm not sure of the value of author and author_email

File: sdk/python/setup.cfg [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+classifiers =
+    Programming Language :: Python :: 3
+    License :: OSI Approved :: MIT License
+    Operating System :: OS Independent
+
+[options]
+package_dir =
+    = oomstore
+packages = find: oomstore
+python_requires = >=3.6
```
Comment (YikSanChan) [inline sdk/python/setup.cfg:21]: lets require >= 3.7

3.6 reaches end of life soon https://endoflife.date/python

File: sdk/python/setup.cfg [hunk 3]
```diff
@@ -0,0 +1,24 @@
+[metadata]
+name = oomstore
+version = 0.0.1
+author = oom
```
Comment (YikSanChan) [inline sdk/python/setup.cfg:4]: author = oom.ai

File: sdk/python/setup.cfg (hunk 1 above)
Comment (YikSanChan) [reply sdk/python/setup.cfg:5]: let's do evan.chanyiksan@gmail.com for now.

Once we setup @oom.ai emails, we can change

**Review by YikSanChan — APPROVED** [review]
lgtm

File: sdk/python/setup.cfg (hunk 2 above)
Comment (lianxmfor) [reply sdk/python/setup.cfg]: done

**Pull request merged**